iter_int_test.go

```diff
@@ -0,0 +1,101 @@
+package jsoniter_test
```

Comment:
Could you move this test case to the *jsoniter_int_test.go* under misc_tests directory?

Reply:
Done. Wasn't sure originally where to place this test 🤷